code/game/gamemodes/blob/blobs/core.dm

```diff
@@ -32,11 +32,7 @@
 
 /obj/structure/blob/core/update_icon()
 	cut_overlays()
-	color = null
-	var/image/I = new('icons/mob/blob.dmi', "blob")
-	if(overmind)
-		I.color = overmind.blob_reagent_datum.color
-	add_overlay(I)
```

Comment:
todo: undo this since it makes the blob core overlay get the blob color and we don't want that. Same for node.